lib/forum-interactions.js

```diff
@@ -61,15 +61,15 @@ export const linkCommentToQuestion = (questionID, commentID, commentList) => {
   }
   return new Promise((resolve, reject) => {
     axios
-      .put(`${API_URL}/questions/${questionID}`, {
```

Comment:
API interactions still need to point to questions I think. this was what I meant when I said it hadnt changed in strapi. comments should work without this change right?

Reply:
That fix might not be required, but the comments links definitely are broken - if you go on someones profile and click the link to the post from the comments tab, it 404s

Reply:
yep. We can keep the links and revert the API call changes